File: Selenium/Project/Activity03.java
package project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity03 {
	WebDriver driver;

	@BeforeClass
	public void beforeClass() {
		driver = new FirefoxDriver();
		driver.get("https://alchemy.hguy.co/crm/index.php?action=Login&module=Users");
	}

	@Test
	public void copyRightText_03() {

		String copyRightText = driver.findElement(By.id("admin_options")).getText();
		System.out.println("The copyright Text is : " + copyRightText);
	}

	@AfterClass

	public void tearDown() {
		driver.quit();
	}

}

File: Selenium/Project/Activity03.py
from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Firefox() as driver:

    driver.get("https://alchemy.hguy.co/crm/index.php?action=Login&module=Users")
    copyrigntText = driver.find_element(By.ID,"admin_options")
    print("The copyRight Text = " ,copyrigntText.text)
    driver.quit()
